caffe2/python/helpers/conv.py

```diff
@@ -18,6 +18,8 @@ def _ConvBase(
     kernel,
     weight_init=None,
     bias_init=None,
+    WeightInitializer=None,
+    BiasInitializer=None,
```

Comment:
Ask @salexspb :) He wanted the CamelCase for the *Initializer to indicate they're classes being passed in, not objects.